specs/functionality.md

```diff
@@ -1,115 +1,129 @@
+        * [1] Unregister camera
     * Browse analytics
-        * Plot cafeteria occupancy
+        * [1] Plot cafeteria occupancy
             * Break by year/month/week/day/hour
-        * Plot review count
+        * [1] Plot review count
             * Break by year/month/week/day/hour
-        * Plot different charts together for correlation analysis
-        * Popularity of dishes based on canteens owners data (optional)
+        * [2] Plot different charts together for correlation analysis
+        * [4] Popularity of dishes based on canteens owners data (optional)
 5. Client app
     * Browse canteens
         * View canteen
-            * Open/closed
-            * Live number of visitors (relatively to the full capacity)
-                * Create widget for mobile devices (optional)
-            * Display predicted occupance (optional)
-            * Has specified food - based on tags (optional)
-            * List menu options (see below)
-            * List historical menus (see below)
-        * Filter canteen list
-            * Open/closed
-            * Minimum rating
-            * Has tags (optional)
-        * Find nearby canteens based on used location (optional on Android)
+            * [2] Open/closed
+            * [1] Live number of visitors (relatively to the full capacity)
+            * [1] List menu options (see below)
+            * [3] Create visitor count widget for mobile devices (optional)
+            * [3] Display predicted occupance (optional)
+            * [3] Has specified food - based on tags (optional)
+            * [3] List historical menus (see below)
+        * [2] Filter canteen list
+            * [2] Open/closed
+            * [2] Minimum rating
+            * [3] Has tags (optional)
+        * [3] Sort canteen list
+            * [3] Rating
+            * [4] Distance (GPS-based)
+        * [3] Find nearby canteens based on used location (Android, optional)
     * Browse menus
-        * Display menu options
-        * Rate option
-        * Get nutrition values (optional)
-    * View most of the statistics data
-        * Plot cafeteria occupancy
-            * Break by year/month/week/day/hour
-            * Correlated to specified menu options (optional)
-        * Plot menu options prices changes (optional)
-        * Plot different cafeterias occupancy comparision
+        * [1] Display menu options
+        * [1] Rate option
+        * [4] Get nutrition values (optional)
+    * View most of the statistic data
+        * Plot relative cafeteria occupancy
+            * [1] Break by year/month/week/day/hour
+            * [4] Correlate to specified menu options (optional)
+        * [2] Plot different cafeterias occupancy comparison
             * Break by year/month/week/day/hour
-    * Provide English version (optional)
-    * Implement accessibility features (optional)
-6. Other prediction pipelines (optional)
+        * [3] Plot menu options prices changes (optional)
```

Comment:
I would change it to [2] and make it maybe not optional. The more maybe simple but elegant elements we have the better our app will look like

Reply:
Yeah, but let's not make everything required. Optional means we can still do it :)